client/controllers/curatorInbox.coffee

```diff
@@ -28,27 +14,35 @@ Template.curatorInbox.onCreated ->
   @calendarState = new ReactiveVar false
   @ready = new ReactiveVar false
   @selectedArticle = false
-  @days = []
+  @dateRange = new ReactiveVar
+    startDate: moment().subtract(1, 'weeks').toDate()
+    endDate: new Date()
   @textFilter = new ReactiveTable.Filter('curator-inbox-article-filter', ['url'])
   @reviewFilter = new ReactiveTable.Filter('curator-inbox-review-filter', ['reviewed'])
-
-  self = @
-
-  Meteor.call 'fetchPromedPosts', 100, null, () ->
-    self.sub = Meteor.subscribe "curatorSources", () ->
-      self.days = createInboxSections()
-      self.ready.set(true)
+  
+  @autorun =>
+    Meteor.call 'fetchPromedPosts', 100, @dateRange.get(), (err) ->
+      if err
+        console.log(err)
+        return toastr.error(err.reason)
+      console.log "source cache updated"
```

Comment:
Do we want to leave this in here on the master branch or was it more for debugging?